friends/pennies/__init__.py

```diff
@@ -0,0 +1,52 @@
+import check50
+import check50.c
+
+@check50.check()
+def exists():
+    """pennies.c exist"""
+    check50.exists("pennies.c")
+
+@check50.check(exists)
+def compiles():
+    """pennies.c compiles"""
+    check50.c.compile("pennies.c", lcs50=True)
+
+@check50.check(compiles)
+def test28days1penny():
+    """28 days, 1 penny on day one yields $2684354.55 """
+    check50.run("./pennies 28 1").stdout("\$2684354.55\n", regex=False).exit(0)
```

Comment:
No need for backslashes in front of "$" if `regex` is false. Ditto for the rest.